A14-channelz.md

```diff
@@ -349,6 +364,7 @@ Channelz exposes four methods to access channelz data:
 * GetTopChannels returns the root channels for channel trees
 * GetServers returns the servers in the system
 * GetChannel returns info about a specific channel
+* GetSubchannel returns info about a specific subchannel
```

Comment:
Weren't we going to change these to `GetChannels` and `GetSubchannels`, plural, so that clients could get more than one of them at once?

Reply:
I thought about it, but it complicates the API.  It's no longer possible to use NOT_FOUND as a return code.